datarobot_batch_scoring/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.13.3'
+__version__ = '1.13.4a1'
```

Comment:
I'd say it's `1.14.0a1`. This functionality is definitely worth bumping _minor_ version.

Reply:
fixed